submissions/description/server_security_misconfiguration/unsafe_file_upload/no_antivirus/template.md

```diff
@@ -0,0 +1,21 @@
+# Unsafe File Upload With No Antivirus
+
+## Overview of the Vulnerability
+
+Unsafe file uploads occur when the web server fails to sufficiently validate the file’s size, type, name, contents, or what restrictions are placed on the file once it has been successfully uploaded. The application fails to validate the size of files that are uploaded. An attacker can serve a Denial of Service (DoS) to the application by uploading large files to the server as there is no file size limit. This may cause outages and slowdowns.
```

Comment:
Unsure why this is using a denial of service over an actual antivirus issue, where malware could be hosted?

Reply:
Apologies, I didn't save the updates I made to the local file properly. Will update this today.

Reply:
Apologies, I didn't save the updates I made to the local file properly. Will update this today.